src/Illuminate/Support/Testing/Fakes/NotificationFake.php

```diff
@@ -270,7 +270,7 @@ public function sendNow($notifiables, $notification, array $channels = null)
 
         foreach ($notifiables as $notifiable) {
             if (! $notification->id) {
-                $notification->id = Str::uuid()->toString();
+                $notification->id = (string) Str::uuid();
```

Comment:
If the required package is not installed this returns a string and because it is not possible to call ->toString() on a string, its better to cast to string as the Uuid class has a __toString() method.